charts/datadog/v2.0.14/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kube-state-metrics
-    version: ~2.2.3
+    version: ~2.3.1
```

Comment:
you need to run `helm dep update` and push the updated charts
e.g, refer to https://github.com/rancher/charts/tree/master/charts/datadog/v1.33.0/charts

Reply:
done